docs/CHANGELOG.md

```diff
@@ -41,6 +41,9 @@ it is attached to. ([#1282])
 [#1281]: https://github.com/amethyst/amethyst/pull/1281
 [#1302]: https://github.com/amethyst/amethyst/pull/1302
 
+* Fixed default system font loading to accept uppercase extension ("TTF") ([#1328])
```

Comment:
Hiya, thanks for this! Please could you move this line to the `Fixed` section above?
Right now it [doesn't render the link correctly](https://github.com/amethyst/amethyst/blob/56c60c235569898eae557edc9ffdc91fd96a0f42/docs/CHANGELOG.md#fixed)